portal-web/docroot/html/taglib/ui/icon_menu/lexicon/start.jsp

```diff
@@ -36,7 +38,7 @@ if (Validator.isNull(icon)) {
 <div class="dropdown lfr-icon-menu <%= cssClass %>" <%= AUIUtil.buildData(data) %>>
 	<c:choose>
 		<c:when test='<%= triggerType.equals("button") %>'>
-			<button aria-expanded="false" aria-haspopup="true" class="btn btn-monospaced btn-secondary dropdown-toggle <%= triggerCssClass %>" id="<%= id %>" title="<%= message %>" type="button">
+			<button aria-expanded="false" aria-haspopup="<%= hasMessage %>" class="btn btn-monospaced btn-secondary direction-<%= direction %> dropdown-toggle <%= triggerCssClass %>" id="<%= id %>" <%= hasMessage ? "title=\"" + message + "\"" : StringPool.BLANK %> type="button">
```

Comment:
I think `aria-haspopup` has to do with the dropdown that can appear when clicking in the button right? not with the tooltip. If so we can leave the `true` there and inline the hasMessage check instead of creating a variable with `Validator.isNotNull(message)` wdyt?

Reply:
Nice catch @victorg1991 

@tinycarol could you take a look?

Reply:
On it, thanks guys

Reply:
Done!